src/components/molecules/Common/AssetModal/AssetContainer/index.tsx

```diff
@@ -70,20 +76,55 @@ const AssetContainer: React.FC<Props> = ({
     { key: "name", label: intl.formatMessage({ defaultMessage: "Alphabetical" }) },
   ];
 
+  const handleRemove = useCallback(() => {
+    if (selectedAssets?.length) {
+      onRemove?.(selectedAssets);
+    }
+  }, [onRemove, selectedAssets]);
+
+  const listRef = useRef<HTMLDivElement>(null);
+  const [containerWidth, setContainerWidth] = useState(0);
+
+  useEffect(() => {
+    if (listRef.current?.offsetWidth) {
+      setContainerWidth(listRef.current?.offsetWidth);
+    }
+  }, [containerWidth]);
```

Comment:
This useEffect can cause an infinite loop. (containerWidth update -> setContainerWidth -> containerWidth update -> ...)